oic.r.cred.raml

```diff
@@ -95,8 +118,19 @@ traits:
         description: The request is invalid.
 
   post:
+
```

Comment:
I just checked the latest version of CR50 and "rowner" was removed from every SVR so I don't think we should be adding it to any SVR, including ACL2.  Am I missing something?